components/build-templates/base/e2e/rolebinding.yaml

```diff
@@ -53,6 +53,19 @@ roleRef:
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: appstudio-pipelines-namespace-creator-rolebinding
+subjects:
```

Comment:
why this change is needed?

Reply:
I applied the namespace-creator to wherever there are configurations related to appstudio-pipelines for uniformity. should those three be removed? @gbenhaim

Reply:
I suggest to wait with this PR for now. I don't think it's needed but maybe I'm missing something.

Reply:
Speaking with the pipeline-services team, it seems like `appstudio-pipelines-runner-rolebinding.yaml` is linking the role (I added in https://github.com/codeready-toolchain/member-operator/pull/529) to the user itself, so I'm not sure if it can work without. I'll wait on the pipelines\build review anyway